packages/react/src/components/Nav/Nav.types.ts

```diff
@@ -101,6 +101,13 @@ export interface INavProps {
    */
   initialSelectedKey?: string;
 
+  /**
```

Comment:
We should try to just pass `focusZoneProps?: IFocusZoneProps` through from a host, allowing settings for any props not controlled by `Nav`.

Reply:
Thank you, quite make sense, it will be much flexible, updated!